src/pages/home/Home.tsx

```diff
@@ -51,24 +51,40 @@ export default observer(() => {
         state.settings.set("appearance:seasonal", !seasonalTheme);
 
     const isDecember = !isTouchscreenDevice && new Date().getMonth() === 11;
+    const isOctober = !isTouchscreenDevice && new Date().getMonth() === 9
     const snowflakes = useMemo(() => {
         const flakes = [];
 
-        // Disable outside of December
-        if (!isDecember) return [];
+        if (isDecember) {
+            for (let i = 0; i < 15; i++) {
+                flakes.push("❄️");
+                flakes.push("❄");
+            }
 
-        for (let i = 0; i < 15; i++) {
-            flakes.push("❄️");
-            flakes.push("❄");
-        }
+            for (let i = 0; i < 2; i++) {
+                flakes.push("🎄");
+                flakes.push("☃️");
+                flakes.push("⛄");
+            }
 
-        for (let i = 0; i < 2; i++) {
-            flakes.push("🎄");
-            flakes.push("☃️");
-            flakes.push("⛄");
+            return flakes;
+        }
+        if (isOctober) {
+            for (let i = 0; i < 15; i++) {
+                flakes.push("🎃");
+                flakes.push("💀");
+            }
+
+            for (let i = 0; i < 2; i++) {
+                flakes.push("👻");
+                flakes.push("⚰️");
+                flakes.push("🕷️");
+            }
+
+            return flakes;
         }
 
-        return flakes;
+        else return [];
```

Comment:
This can remain as is - if `isDecember` and `isOctober` are both false, the `flakes` array will still be empty

Reply:
Alright, I just changed `if` to `else if` so that `else` applies to both cases. (should fix that little issue)

Reply:
I think you slightly misunderstood my comment - I meant that none of the `else`s are required